src/db.rs

```diff
@@ -0,0 +1,14 @@
+use crate::env;
+use sqlx::PgPool;
+
+fn get_db_url() -> String {
+    env::get_env_var_unsafe("DATABASE_URL")
+}
+
+pub async fn get_db_pool() -> PgPool {
+    let db_url = get_db_url();
+    print!("Connecting to DB: {}", db_url);
```

Comment:
please use macros from `tracing`.
```
use tracing::debug;
debug!(db_url, "connecting to DB");
// or if you prefer formatting / interpolation
debug!("connecting to DB {}", db_url);

Reply:
Good point.  Ended up completely removing this though, since it was liable to log out sensitive data (like password contained in the database url):
https://github.com/ultrasoundmoney/ultra-sound-api/pull/1/commits/f992bd358613b2565cd1425ead51c9c1a7bf7c5d